### DISCUSSION TITLE
Any plans to merge and pull request to original pure-data repository?

### DISCUSSION THREAD

**Discussion by MrLinDowsMac** [sebshader/pdnext#9] [Q&A] [answered]
Well, you added color to canvas to vainilla, is a feature that i would like to have in original vainilla-pd. How about making a pull request to the original repository?

**Comment by sebshader** [comment, marked as answer]
I did once
https://github.com/pure-data/pure-data/pull/196
but I guess it has to be modified for the newest version

**Comment by MrLinDowsMac** [comment]
Perhaps, you could rebase this branch to the latest commit of master (after pulling of course) and fix the complain of Codacy, and then try to pull request again.

**Comment by sebshader** [comment]
ok I think I got it updated. I also have a branch with inlet/outlet highlighting but I'm not sure what I should do with it..
